.travis.yml

```diff
@@ -66,6 +66,44 @@ after_script:
   - zeus upload -t "text/xml+checkstyle" .artifacts/*checkstyle.xml
   - zeus upload -t "application/webpack-stats+json" .artifacts/*webpack-stats.json
 
+base_postgres: &postgres_default
```

Comment:
Dammit @wedamija, going hard on the YAML here.

Reply:
Love it